core/log/log_allocator.odin

```diff
@@ -80,6 +80,13 @@ log_allocator_proc :: proc(allocator_data: rawptr, mode: runtime.Allocator_Mode,
 				la.prefix, padding, old_memory, old_size, size, alignment,
 				location = location,
 			)
+		case .Resize_Non_Zeroed:
```

Comment:
The log allocator has been changed on master to allow for human readable byte format and to use a stack buffer instead of temp allocating to prevent recursion when logging the temp allocator. Could you apply these changes to this case too?

Reply:
yep, will do